### PR TITLE
Fix compatibility issue in Makefie on x86_64 ubuntu 20.04

### PR DESCRIPTION
**Pull request by lida2003**
a) Support ubuntu 20.04 x86_64 platform
b) Support orginal raspberry Pi RPI 3B+, should work with other PIs

### CODE REVIEW

**Review by whstudio123 — APPROVED** [review]
I have checked that it could run without problem.

**Comment by jeanlemotan** [comment]
Thanks a lot for the support

File: gs/src/imgui_impl_opengl3.cpp [hunk 1]
```diff
@@ -270,7 +270,11 @@ void    ImGui_ImplOpenGL3_RenderDrawData(ImDrawData* draw_data, bool rotate)
                     }
 
                     // Bind texture, Draw
+#if defined(RASPBERRY_PI)
                     uint32_t texId = (uint32_t)pcmd->TextureId;
+#else
+                    uint64_t texId = (uint64_t)pcmd->TextureId;
```
Comment (jeanlemotan) [inline gs/src/imgui_impl_opengl3.cpp:276]: Could we use uint64_t for the pi as well? Or size_t?

Comment (lida2003) [reply gs/src/imgui_impl_opengl3.cpp:276]: TextureID comes from imgui  struct ImDrawCmd. I think it's the library definition.

Comment (lida2003) [reply gs/src/imgui_impl_opengl3.cpp:274]: Please check [Fix void * type cast issue on 32/64bit systems](https://github.com/jeanlemotan/esp32-cam-fpv/pull/25/commits/47efa1a868e0acc25ce7a8deafda53fff4beb14e)

Comment (lida2003) [reply gs/src/imgui_impl_opengl3.cpp]: intptr_t  is a good idea.

**Pull request merged**